src/kibana/utils/_mixins_chainable.js

```diff
@@ -250,11 +262,18 @@ define(function (require) {
     },
 
     /**
-     * Shortcut for the simple version of _.deepGet
+     * Shortcut for the simple version of _.deepGet with support for default
+     * values added
+     *
+     * @param {obj} any - the value to read from
+     * @param {string|array} path - the location of the value to return as
+     *                           a dot-notated string or array of keys.
+     * @param {any} def - when the value is null or undefined return this instead
      * @return {any}
      */
-    get: function (obj, path) {
-      return _.deepGet(obj, path);
+    get: function (obj, path, def) {
+      var val = _.deepGet(obj, path);
+      return (val == null && def != null) ? def : val;
```

Comment:
Can we convert these to triple equals as well?

Maybe we could add a function to chainable mixins to check; ```val === undefined || val === null;```